temporian/test/BUILD

```diff
@@ -66,16 +66,3 @@ py_test(
         "//temporian",
     ],
 )
-
-py_test(
-    name = "public_api_test",
-    srcs = ["public_api_test.py"],
-    data = [
-        "//docs:reference",
-    ],
-    srcs_version = "PY3",
-    deps = [
-        # already_there/absl/testing:absltest
-        "//temporian",
-    ],
-)
```

Comment:
👀

Reply:
I moved it in docs/BUILD (it makes my life a bit easier with the sync with internal version).